python/test/unit/language/test_core.py

```diff
@@ -2484,9 +2473,6 @@ def _welford_combine(mean_1, m2_1, weight_1, mean_2, m2_2, weight_2):
 @pytest.mark.parametrize("op", ["sum", "max"])
 @pytest.mark.parametrize("first_axis", [0, 1])
 def test_chain_reduce(M, N, src_layout, op, device, first_axis):
-    if is_hip():
```

Comment:
Are all tests passing in chain_reduce?

Reply:
Yes, all passed. But two cases are flaky with `[1, THREADS_PER_WARP]` for the third layout.

Reply:
If this is the case, they might cause CI to fail.
Can we skip those two cases for now?

Reply:
We are not using the setting [threads_per_warp//32, 32] for the third layout now, so all test can always. There is no flaky problem for now. 

Let us see, if we saw a problem, we can skip those tests.